Caboodle/Browser/Browser.shared.enums.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Caboodle
+{
+    public enum BrowserLaunchingType
+    {
+        UriLauncher,
+        SystemBrowser
```

Comment:
This one is really something that my gut doesn't like... Maybe rather that talking about the browser, we could name this:

```csharp
enum BrowserLaunchMode {
    ExternalBrowser
    SystemPreferred
}
```